packages/core/src/sdk.ts

```diff
@@ -48,15 +47,15 @@ export function setCurrentClient(client: Client): void {
 }
 
 /**
- * Unfortunately, we still have to manually bind the client to the "hub" property set on the global
+ * Unfortunately, we still have to manually bind the client to the "stack" property set on the global
  * Sentry carrier object. This is because certain scripts (e.g. our loader script) obtain
- * the client via `window.__SENTRY__.hub.getClient()`.
+ * the client via `window.__SENTRY__[version].stack.getClient()`.
```

Comment:
ah wait, I see now - this just sets this on the stack 🤔 I guess that's ok for now...

Reply:
I mean we could also just set a flag, something like `window.__SENTRY__[version].sdkInitialized = true` instead, right? At least for the loader that's enough I think, since we only check if the client exists or not 🤔 

Spotlight currently accesses `window.__SENTRY__.acs` which I still need to change to  `window.__SENTRY__[version].acs` once this PR lands.

Reply:
could we just check if `window.__SENTRY__[version]` exists, actually? This will only be set if the SDK was initialized I guess..? 😅